### ISSUE TITLE
[Bug]: user_provided don't request openai token on new Ui

### ISSUE THREAD

**Issue by Airkek** [danny-avila/LibreChat#1253]
### What happened?

When openai token set to 'user_provided', token request popup is not shown to user

### Steps to Reproduce

1) set .env:
```
OPENAI_API_KEY=user_provided
```

2) Revoke tokens on Ui
3) Say "Hi" to OpenAI

### Relevant log output

```shell
Dec 01 10:16:21 kds start.sh[67429]: {
Dec 01 10:16:21 kds start.sh[67429]:   text: 'hi',
Dec 01 10:16:21 kds start.sh[67429]:   conversationId: null,
Dec 01 10:16:21 kds start.sh[67429]:   endpointOption: {
Dec 01 10:16:21 kds start.sh[67429]:     endpoint: 'openAI',
Dec 01 10:16:21 kds start.sh[67429]:     chatGptLabel: null,
Dec 01 10:16:21 kds start.sh[67429]:     promptPrefix: null,
Dec 01 10:16:21 kds start.sh[67429]:     modelOptions: {
Dec 01 10:16:21 kds start.sh[67429]:       model: 'gpt-4',
Dec 01 10:16:21 kds start.sh[67429]:       temperature: 1,
Dec 01 10:16:21 kds start.sh[67429]:       top_p: 1,
Dec 01 10:16:21 kds start.sh[67429]:       presence_penalty: 0,
Dec 01 10:16:21 kds start.sh[67429]:       frequency_penalty: 0
Dec 01 10:16:21 kds start.sh[67429]:     }
Dec 01 10:16:21 kds start.sh[67429]:   }
Dec 01 10:16:21 kds start.sh[67429]: }
Dec 01 10:16:21 kds start.sh[67429]: Error: API key not provided.
Dec 01 10:16:21 kds start.sh[67429]:     at initializeClient (/home/keijia/LibreChat/api/server/routes/endpoints/openAI/init>
Dec 01 10:16:21 kds start.sh[67429]:     at /home/keijia/LibreChat/api/server/routes/ask/openAI.js:95:30
Dec 01 10:16:21 kds start.sh[67429]:     at Layer.handle [as handle_request] (/home/keijia/LibreChat/node_modules/express/li>
Dec 01 10:16:21 kds start.sh[67429]:     at next (/home/keijia/LibreChat/node_modules/express/lib/router/route.js:144:13)
Dec 01 10:16:21 kds start.sh[67429]:     at setHeaders (/home/keijia/LibreChat/api/server/middleware/setHeaders.js:9:3)
Dec 01 10:16:21 kds start.sh[67429]:     at Layer.handle [as handle_request] (/home/keijia/LibreChat/node_modules/express/li>
Dec 01 10:16:21 kds start.sh[67429]:     at next (/home/keijia/LibreChat/node_modules/express/lib/router/route.js:144:13)
Dec 01 10:16:21 kds start.sh[67429]:     at buildEndpointOption (/home/keijia/LibreChat/api/server/middleware/buildEndpointO>
Dec 01 10:16:21 kds start.sh[67429]:     at Layer.handle [as handle_request] (/home/keijia/LibreChat/node_modules/express/li>
Dec 01 10:16:21 kds start.sh[67429]:     at next (/home/keijia/LibreChat/node_modules/express/lib/router/route.js:144:13)
lines 957-1000/1000 (END)
```

### Screenshots

![image](https://github.com/danny-avila/LibreChat/assets/45855057/00d0e18c-ed0b-445c-b63a-27343e305085)

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
will look into it later today, is it still possible to manually set it?

**Comment by Airkek** [comment]
> is it still possible to manually set it?

Seems like no, at least I can't find this ability in the Ui

**Comment by danny-avila** [comment]
Fixed in #1281
